.github/workflows/scripts/kinds/verify-kinds.go

```diff
@@ -37,31 +40,17 @@ const (
 // If kind names are given as parameters, the script will make the above actions only for the
 // given kinds.
 func main() {
-	var kindArgs []string
-	if len(os.Args) > 1 {
-		kindArgs = os.Args[1:]
-	}
-
```

Comment:
I decided to delete the possibility to run the script for only a subset of kinds as this was implemented as a solution to run the script locally without creating a GH token but it's actually better to run it for all the kinds if possible.